docs/core/README.md

```diff
@@ -9,19 +9,20 @@ parent:
 This repository contains reference documentation on the core concepts of the Cosmos SDK.
 
 1. [`BaseApp`](./baseapp.md)
-1. [Transaction](./transactions.md)
-1. [Context](./context.md)
-1. [Node Client](./node.md)
-1. [Store](./store.md)
-1. [Encoding](./encoding.md)
-1. [gRPC, REST and Tendermint Endpoints](./grpc_rest.md)
-1. [Command-Line Interface](./cli.md)
-1. [Events](./events.md)
-1. [Telemetry](./telemetry.md)
-1. [Object-Capabilities](./ocap.md)
-1. [RunTx recovery middleware](./runtx_middleware.md)
-1. [Simulation](./simulation.md)
-1. [Protobuf documentation](./proto-docs.md)
+2. [Transaction](./transactions.md)
+3. [Context](./context.md)
+4. [Node Client](./node.md)
+5. [Store](./store.md)
+6. [Encoding](./encoding.md)
+7. [gRPC, REST and Tendermint Endpoints](./grpc_rest.md)
+8. [Command-Line Interface](./cli.md)
+9. [Events](./events.md)
+10. [Telemetry](./telemetry.md)
+11. [Object-Capabilities](./ocap.md)
+12. [RunTx recovery middleware](./runtx_middleware.md)
+13. [Simulation](./simulation.md)
+14. [Protobuf documentation](./proto-docs.md)
+15. [Upgrades](./upgrade.md)
```

Comment:
```suggestion
15. [In-Place Store Migrations](./upgrade.md)
```

btw, using `1.` is a markdown trick to show auto-incrementing numbers, but we can also be explicit here

Reply:
ah i see. i felt it was a bid odd, since entering a new line increments the number for you in certain editors. thought might as well take advantage of that 🤷🏼